viewer/src/public/RevealManager.ts

```diff
@@ -19,6 +19,7 @@ import { PotreeGroupWrapper } from '@/datamodels/pointcloud/PotreeGroupWrapper';
 import { PotreeNodeWrapper } from '@/datamodels/pointcloud/PotreeNodeWrapper';
 import { RenderMode } from '@/datamodels/cad/rendering/RenderMode';
 import { EffectRenderManager } from '@/datamodels/cad/rendering/EffectRenderManager';
+import { SupportedModelTypes } from '@/public/migration/types';
```

Comment:
It would be better if you made the types part of utilities or datamodels. I believe we try and limit the import dependency paths. (I think the rules were:
datamodels can import from utilities but not the other way around.
public can import from utilities and datamodels but not the other way around.

Reply:
@j-bjorne Do you mean just move all the types under utilities? I don't think it's actually a good idea because we will lose that type in the documentation then. Now it's [documented here](https://cognitedata.github.io/reveal/docs/API%20Reference#enums_migration_types_supportedmodeltypesmd)

Utilities folder is ignored from docs. Because otherwise there would be tons of stuff from utils that also will pop up into the API reference.

Reply:
Then add it under datamodel/common or something, or datamodel/core some folder to indicate its shared state between datamodels I guess. I just know we made a choice a few weeks back and determined that we would have import restrictions, and I believe this import is breaking that restriction.

`src/datamodels/{common/core/shared}/types.ts` or something could contain this information instead.

Reply:
Then add it under datamodel/common or something, or datamodel/core some folder to indicate its shared state between datamodels I guess. I just know we made a choice a few weeks back and determined that we would have import restrictions, and I believe this import is breaking that restriction.

src/datamodels/{common/core/shared}/types.ts or something could contain this information instead.

Reply:
@j-bjorne OK, I moved `SupportedModelTypes` under datamodels, but had to reexport them in public/types to keep in the API reference.